src/App.js

```diff
@@ -1,7 +1,29 @@
+import MockmanEs from "mockman-js";
+import { Route, Routes } from "react-router-dom";
 import "./App.css";
+import {
+  Login,
+  Signup,
+  Products,
+  Wishlist,
+  Cart,
+  Home,
+} from "./barrelexport/Pageutil";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
```

Comment:
you can create a separate component for routes , the App component should always be minimal and clean

Reply:
ok got it, thanks.